.github/workflows/ci-windows.yml

```diff
@@ -104,3 +102,59 @@ jobs:
         with:
           path: ${{ env.CACHE_DIR }}
           key: windows-build-test-cpp-asserts-v1-${{ github.sha }}-${{ github.event.repository.updated_at }}
+
+  test_windows:
+    name: E2E Test windows
+    runs-on: windows-phoenix
+    needs: build_and_ctest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: "Checking out repository" # for test scripts
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+        with:
+          submodules: false # not required for testbench
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: windows_x86_64_release_packages
+
+      - name: Extract artifact
+        run: |
+          mkdir iree-install
+          tar -xf iree-dist-windows.tar -C iree-install
+          bash build_tools/download_peano.sh
+
+      - name: Create venv and install dependencies
+        run: |
+          python -m venv .venv
+          source .venv/Scripts/activate
+          pip install -r tests/matmul/requirements.txt
+
+      - name: E2E correctness matmul test
+        run: |
+          source .venv/Scripts/activate
+          export XILINX_XRT=/c/Xilinx/XRT
```

Comment:
On windows OS, all applications are supposed to uses xrt_coreutil.dll from c:\windows\system32. But this line make the CI computer use the xrt_coreutil.dll from /c/Xilinx/XRT. 
This may cause all our tests get a different test result from a system deployed to general public

Reply:
Agree with Dezhi's statement.  On Windows XILINX_XRT should not be set when running tests.

Reply:
> On Windows XILINX_XRT should not be set when running tests.

As far as I am aware, XRT is open source and thus we are free to use it according to how it best suits our needs. So we have our usecase and implementation and you have yours. If you feel so strongly, you are free to refactor your code so that it doesn't expose these env variables (and I am free to fork).

Reply:
@makslevental ,
What version of NPU KMD driver is installed on the windows computer used for CI?

Reply:
Please stop commenting on this PR. It is now 3 months since it was closed. If you have some current issue then create a new issue about it.